src/components/@core/navigation-menu/common/pages-menu-items.tsx

```diff
@@ -0,0 +1,67 @@
+import { ChevronDownIcon } from "@chakra-ui/icons";
+import { Box, Link, Menu, MenuButton, useBreakpointValue } from "@chakra-ui/react";
+import LocalLink from "@components/@core/local-link";
+import React from "react";
+
+import SubMenu from "./sub-menu";
+
+const SimpleLink = ({ children, to, params }) => (
```

Comment:
Don't we already have some generic link components in the codebase?.
If the existing one's don't satisfy the requirements, extract out this `SimpleLink` component inside `@core` directory (like the rest of the link components) and import it here. we will get this as a reuasble component for other features if needed.

Reply:
Its unique to navbar , we can have this here itself.